### PR TITLE
Build and IDE guides

### PR DESCRIPTION
**Pull request by pjcozzi**
This breaks the old Contributor's Guide into separate guides for building Cesium and setting up an IDE.

The "new" [Build Guide](https://github.com/AnalyticalGraphicsInc/cesium/blob/build-guide-and-friends/Documentation/Contributors/BuildGuide/README.md) is the same as before just with these tweaks: f56ac390bce1ae7075cb3b857426e1fc1f42fae0

Does anyone have a one-liner for:
> Delete all branches expect `master`

**TODO**
* [x] WebStorm and Eclipse IDE Guides
* [x] Update wiki to point to new guides

### CODE REVIEW

File: Documentation/Contributors/BuildGuide/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+* Setup Git if it isn't already ([link](https://help.github.com/articles/set-up-git/#platform-all)).
+   * Make sure your SSH keys are configured ([linux](https://help.github.com/articles/generating-ssh-keys#platform-linux) | [mac](https://help.github.com/articles/generating-ssh-keys#platform-mac) | [windows](https://help.github.com/articles/generating-ssh-keys#platform-windows)).
+   * Double-check your settings for name and email: `git config --get-regexp user.*`.
+   * Recommended Git settings:
+      * `git config --global pull.rebase preserve` - when pulling remote changes, rebase your local changes on top of the remote changes, to avoid unnecessary merge commits.
+      * `git config --global fetch.prune true` - when fetching remote changes, remove any remote branches that no longer exist on the remote.
+* Have commit access to cesium?
+   * No
+      * Fork [cesium](https://github.com/AnalyticalGraphicsInc/cesium).
+      * Delete all branches expect `master`.
```
Comment (mramato) [inline Documentation/Contributors/BuildGuide/README.md]: Typo `expect` -> `except`

**Comment by mramato** [comment]
> Delete all branches expect master

There's no good cross-platform one-liner for this, I recommend suggesting people use the GitHub GUI by going to the all branches page for the fork (i.e. https://github.com/AnalyticalGraphicsInc/cesium/branches/all for cesium itself).

**Comment by mramato** [comment]
Seems like the guide would be harder to maintain if it's broken up, but I'm okay with it if everyone else is.

**Comment by pjcozzi** [comment]
This is ready.

Added the [WebStorm Guide](https://github.com/AnalyticalGraphicsInc/cesium/blob/build-guide-and-friends/Documentation/Contributors/WebStormGuide/README.md) and [Eclipse Guide](https://github.com/AnalyticalGraphicsInc/cesium/blob/build-guide-and-friends/Documentation/Contributors/EclipseGuide/README.md).

These are largely untouched except for minor wording tweaks.

**Comment by pjcozzi** [comment]
This is ready.

**Comment by pjcozzi** [comment]
Made a few tweaks to CONTRIBUTING.md in this branch.  I don't plan to change anything else in this PR.

**Comment by hpinkos** [comment]
Looks good!  Thanks @pjcozzi

**Pull request merged**